binary_and_decimal.rb

```diff
@@ -13,9 +23,28 @@ def binary_to_decimal
 #       You may use array indexing to retrieve one element at a time,
 #       compare individual elements with each other and
 #       you may retrieve the length of an array.
-def array_equals(array1, array2)
-  puts "Not implemented"
-  return true
+def array_equals(one, two)
+
+  if one.length != two.length
+    false
+  else
+      index = 0
+    one.length.times do |compare|
+      if one[index] == two[index]
+        true
+      else
+        false
+        exit
```

Comment:
This should be return instead of exit. `return` returns from the method, `break` breaks out of a loop and `exit` ends the program.